content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Supported releases"
+linkTitle: "Supported releases"
```

Comment:
suggestion: title casing again; current titles of pages follow title casing

```suggestion
title: "Supported Releases"
linkTitle: "Supported Releases"
```

Reply:
Arghhh I just realized that the WHOLE website uses the Title Case convention for sections, which is (opinion!) a typographic mistake ([ref](https://uxplanet.org/why-letter-casing-is-important-to-consider-during-design-decisions-50402acd0a4e)) in the context of sections:

> Title Case is Inconsistent, Sometimes Even Incongruous. It Makes Sentences Little Difficult To Read As It Makes You Stop and Process Every Single Word. As If Giving Unnecessary Emphasis To Words. I Mean Why?

For the sake of consistency, I'll accept your suggestion 😇